src/client/src/ui/blotter/model/blotterModel.js

```diff
@@ -69,6 +69,9 @@ export default class BlotterModel extends ModelBase {
               else {
                 this.trades.unshift(trade);
               }
+              if (!trade.isStateOfTheWorld){
+                this._blotterService.createNotification(trade);
```

Comment:
openFin is injected in services that use it, we don't inject it in the models/views directly. can you suggest a way to access/call openFin directly from the model?